$NetBSD: patch-ab,v 1.1.1.1 1999/12/02 20:46:35 rh Exp $

Perl's pod2man complains of missing "DESCRIPTION", so add one.

--- EmbperlD.pod.dist	Wed Nov 17 07:40:48 1999
+++ EmbperlD.pod	Thu Dec  2 19:45:01 1999
@@ -8,6 +8,8 @@
 
 I<Embperl> ist ein Perlmodul welches es erlaubt beliebigen Perlcode in HTML Dokumente einzufügen.
 
+=head1 DESCRIPTION
+
 =head1 Aufrufen/Konfigurieren von Embperl
 
 =head2 mod_perl Konfiguration
